components/Animations/src/Xaml/AnimationSet.cs

```diff
@@ -51,7 +51,7 @@ public sealed class AnimationSet : DependencyObjectCollection
     /// <summary>
     /// Gets or sets the weak reference to the parent that owns the current animation collection.
     /// </summary>
-    internal WeakReference<UIElement>? ParentReference { get; set; }
+    public WeakReference<UIElement>? ParentReference { get; set; }
```

Comment:
@Arlodotexe Not sure if this is the right thing to do here.. weird how the current toolkit wasn't complaining about this being inaccessible?

Reply:
We might be missing an `InternalsVisibleTo` that the toolkit had before?

Reply:
`StartAnimationActivity.cs` (L66) and `StopAnimationActivity.cs` (L67)

Reply:
Looks like we had an `InternalsVisibleTo` on the Animations package. Let's try to avoid changing the API surface, we can add this instead.

![image](https://github.com/CommunityToolkit/Windows/assets/9384894/f8cfb355-3bff-4dc5-a4ec-605b3930d0b0)